R/phys2eventtime.R

```diff
@@ -20,7 +20,7 @@ phys2eventtime <- function(z, events, width=10) {
  if (is.null(ncol(z))) {
    stop(paste("'z' should be of class zoo/xts with at least one column. Use '[' with drop = FALSE"))
  }
- if (!any(class(events$when) %in% c("POSIXt", "Date"))) {
+ if (!any(class(events$when) %in% c("POSIXt", "Date", "numeric"))) {
```

Comment:
I feel unless there is a need for real number index we should just stick with `integer` as it is much more efficient.

Reply:
`class(events$when)` on a `data.frame` with an integer `when` column returned `numeric`, so that's what I put down.  It seems like if I put `integer` there, it wouldn't work?

```r
> dates.data.frame <- data.frame(
+     name = c("variable.one", "variable.two"),
+     when = c(3, 4))
> class(dates.data.frame$when)
[1] "numeric"
```

Reply:
`1` is always numeric in R, do: `class(1L)`.

Reply:
Aha!  Thanks for the tip :D